src/pages/volunteer/index.tsx

```diff
@@ -114,24 +114,27 @@ export default () => {
       />
       <Spacer heightDesktop="120px" heightMobile="0" />
       <TwoParagraphBlock
-        title="What it’s like to volunteer at ShelterTech"
-        paragraph1="No matter what role you play, the essential attitude you need is a deep belief in digital equity for all people."
+        title="Volunteering at ShelterTech"
+        paragraph1=""
         paragraph2={
           <>
             <p>We ask that you consider these three things before joining:</p>
             <p>
-              &ndash; You are willing to volunteer at least three hours each
-              week.
+              &bull; You are willing to volunteer at least three hours each week
```

Comment:
It looks like this was already the case before your changes here, but it'd be better to use the markup for an actual unordered list (`<ul>`) here instead of inserting a character entity for a bullet.